### ISSUE TITLE
Question about training

### ISSUE THREAD

**Issue by jyang68qq** [InhwanBae/ENet-SAD_Pytorch#9]
Hi, 
I was digging into your code recently. Nice work!

Just had question about your CULane training dataset. It seems that you didnt use txt which contains axis for lanes in the given dataset. 

Could your give a little hint why this would work?

Thanks!

**Comment by InhwanBae** [comment]
Hi @jyang68qq 

Thanks for your interest in my implementation. In the [original paper(Section 3.1: Self Attention Distillation)](https://openaccess.thecvf.com/content_ICCV_2019/papers/Hou_Learning_Lightweight_Lane_Detection_CNNs_by_Self_Attention_Distillation_ICCV_2019_paper.pdf), the total loss is comprised of four terms. here, the first **_L_seg_** uses cross-entropy loss, which means the author approaches this lane detection problem as a semantic segmentation task. After predicting 4 lane areas through semantic segmentation, the small network **_P1_** predicts whether each lane really exists or not with 4 binary values.

Following the original paper(Section 3.3: Architecture Design), I use the **_laneseg_label_** provided in the CULane dataset.

I hope this answers your questions.

**Comment by InhwanBae** [comment]
Thank you for your interest. I'm closing this issue for now. Please feel free to open another issue for other questions.
